x-pack/plugins/cloud_security_posture/server/saved_objects/migrations/csp_rule_template.ts

```diff
@@ -28,7 +28,7 @@ function migrateCspRuleTemplatesToV840(
       muted,
       metadata: {
         ...metadata,
-        benchmark: { ...benchmark, id: 'cis_k8s' },
+        benchmark: { ...benchmark, id: 'cis_k8s', posture_type: 'kspm' },
```

Comment:
@JordanSh - i think we need to add `rule_number` here too?

Reply:
I don't think so, since we can't give the same value to all the rules.
that leaves us with either an empty string to prevent calling functions on undefined and such. but i think handling the undefined value is a bit better since atleast we will handle missing values in some way. with an empty string we most likely wont have any workaround and just leave stuff as they are.

call me if we need to discuss it further, I'm also tagging @kfirpeled to hear his thoughts